### PR TITLE
Fix docker_image's build.args

### LINKED ISSUE #455: docker_image build args failing
##### SUMMARY

Hello,

The docker_image build args doesn't seem to work.

With this simple playbook:

```yaml
---
- name: Test
  hosts: localhost
  any_errors_fatal: true
  tasks:
    - docker_image:
        name: test
        state: present
        source: build
        build:
          path: .
          dockerfile: "Dockerfile"
          pull: no
          args:
            test: centos:7
```
And a very small Dockerfile:
```yaml
ARG test
FROM $test
RUN echo $test
```
I get the following error message: `Error building test - code: None, message: base name ($test) should not be blank, logs: ['Step 1/3 : ARG test', '', 'Step 2/3 : FROM $test', '']`.

I'm on python 3.8.10, ansible core 2.13.3 and community.docker 3.0.1.

Many thanks in advance for your help or guidance.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
- docker_image

##### ANSIBLE VERSION

```
ansible [core 2.13.3]
  config file = None
  configured module search path = ['/home/vagrant/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/vagrant/.local/pipx/venvs/ansible-core/lib/python3.8/site-packages/ansible
  ansible collection location = /home/vagrant/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/vagrant/.local/bin/ansible
  python version = 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
```

##### COLLECTION VERSION

```
Collection        Version
----------------- -------
community.general 2.5.8
community.docker 3.0.1
```

##### OS / ENVIRONMENT

I discovered this issue on Ubuntu 20.04 (on a vagrant box: https://app.vagrantup.com/bento/boxes/ubuntu-20.04).

##### STEPS TO REPRODUCE

See summary for the reproducible test case.

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fixes #455.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
docker_image

### CODE REVIEW

**Comment by gotmax23** [comment]
Why do the tests have `ignore_errors: yes`? Errors should get caught by

```
- assert:
    that:
    - buildargs_1 is changed
    - buildargs_2 is not failed and buildargs_2 is not changed
```

so I'll approve this.

**Review by gotmax23 — APPROVED** [review]

File: tests/integration/targets/docker_image/templates/ArgsDockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Ansible Project
```
Comment (gotmax23) [inline tests/integration/targets/docker_image/templates/ArgsDockerfile:1]: Do you want to replace `Ansible Project` with your own name?

**Comment by felixfontein** [comment]
> Why do the tests have `ignore_errors: yes`? Errors should get caught by

I was wondering about that as well. I think they can just be dropped, but I wanted to do that in another PR.

File: tests/integration/targets/docker_image/templates/ArgsDockerfile (hunk 1 above)
Comment (felixfontein) [reply tests/integration/targets/docker_image/templates/ArgsDockerfile:1]: Yeah, why not. (Not sure whether this file actually meets the threshold of originality...)
```suggestion
# Copyright (c) 2022, Felix Fontein
```

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Pull request merged**